tests/unit/Middleware/RecordTransactionTest.php

```diff
@@ -47,12 +47,15 @@ protected function _before()
         $this->response->headers = new ResponseHeaderBag();
     }
 
-    protected function createMiddlewareInstance(bool $use_route_uri): void
+    protected function createMiddlewareInstance(bool $use_route_uri, ?string $ignore_patterns): void
     {
         $this->config = Mockery::mock(Config::class);
         $this->config->shouldReceive('get')
             ->with('elastic-apm-laravel.transactions.useRouteUri')
             ->andReturn($use_route_uri);
+        $this->config->shouldReceive('get')
+            ->with('elastic-apm-laravel.transactions.ignorePatterns')
```

Comment:
missing my favourite Mockery method:

```suggestion
            ->with('elastic-apm-laravel.transactions.ignorePatterns')
            ->once()
```